newsService/src/main/java/com/kovan/app/service/TestService.java

```diff
@@ -0,0 +1,39 @@
+package com.kovan.app.service;
+
+
+import com.kovan.api.model.TestRequest;
+import com.kovan.dto.TestDto;
+import com.kovan.service.TestRepositoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+
+
+@Service
+public class TestService {
+
+
+    @Autowired
+    private TestRepositoryService service;
+
+
+    public TestDto addData(TestRequest testRequest) {
+
+        if (testRequest == null) {
```

Comment:
Don't check null like this use StringUtils

Reply:
Issue resolved.